src/core-styling/functions/_validation.scss

```diff
@@ -0,0 +1,9 @@
+$_is-test: false !default;
+
+@function error($message, $capture: $_is-test) {
```

Comment:
Previously was `_error`, but that enforces a privacy when `@use`d that we don't want in this case.